driver.go

```diff
@@ -295,21 +297,23 @@ func mount(accountName, accountKey, mountpoint string, options VolumeOptions) er
 		options.GID = "0"
 	}
 	mount := fmt.Sprintf("//%s.file.core.windows.net/%s", accountName, options.Share)
-	opts := fmt.Sprintf(
-		"vers=3.0,username=%s,password=%s,dir_mode=%s,file_mode=%s,uid=%s,gid=%s",
-		accountName,
-		accountKey,
-		options.DirMode,
-		options.FileMode,
-		options.UID,
-		options.GID,
-	)
+	opts = append(opts, "vers=3.0")
```

Comment:
I am not sure append is a good way to construct templates. I think the old version was more readable?